backend/models/taskModel.js

```diff
@@ -2,10 +2,16 @@ const mongoose = require('mongoose');
 
 // TODO: update this schema to contain essential properties
 const taskSchema = new mongoose.Schema({
-    taskName: {
-        type: String,
-        required: true
-    }
+  name: {
+    type: String,
+    required: true,
+  },
+  isDone: {
+    type: Boolean,
+    required: true,
+  },
+  todoListId: { type: mongoose.Schema.Types.ObjectId, ref: 'TodoList' },
+  userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
```

Comment:
I think that there are more fields to be added, such as category, time, prioritization etc. I don't know where else we would like to store it.

Reply:
Yes, you're right. When I created this, I hadn't seen your scoring function, and I didn't know exactly what data should be included :)

Reply:
I think that once you merge your function, I will also add a 'score' property to the 'taskModel' so that when adding a task, it calculates the score immediately and adds it to the database.